nsxt/resource_nsxt_vpc_nat_rule.go

```diff
@@ -76,6 +76,7 @@ var policyVpcNatRuleSchema = map[string]*metadata.ExtendedSchema{
 			Type:         schema.TypeString,
 			ValidateFunc: validation.StringInSlice(policyVpcNatRuleActionValues, false),
 			Optional:     true,
```

Comment:
It can't be both optional and required, I think

Reply:
Why? It's an Heisen-Attribute!